judge/src/tested/languages/kotlin/templates/function.mako

```diff
@@ -0,0 +1,23 @@
+## This translates a function call to Kotlin.
+<%! from tested.serialisation import FunctionType, NamedArgument, Expression %>\
+<%! from tested.utils import get_args %>\
+<%page args="function" />\
+% if function.type == FunctionType.NAMESPACE:
+    ${function.namespace}?.\
```

Comment:
I will change it to the non-null assertion operator, just removing it won't work because the object variable contains a null value on initalisation (try-catch for the variable could fail) and the variable must be initialised before use.